inc/vite.php

```
@@ -10,22 +10,34 @@
+        }
+        return $this->dist_path;
+    }
 
-        $this->dist_path = get_theme_file_path('/dist');
-        $this->dist_url  = get_template_directory_uri() . '/dist';
+    private function get_dist_url() {
+        if ($this->dist_url === null) {
+            $this->dist_url = get_template_directory_uri() . '/dist';
+        }
+        return $this->dist_url;
     }
```

PHPMD (2.15.0)

[error] The method get_dist_url is not named in camelCase. (undefined)

(CamelCaseMethodName)